gizmo.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+
+
+class Gizmo:
+    def __init__(self, name):
+        self.name = name
+
+    def speak(self):
+        print(self.name)
+
+
+def hello(name, country="Finland"):
+    print(f"Hello {name}, how are things in {country}?")
+
+
+def spell(word):
+    print(".".join(word))
```

Comment:
I never realized you wouldn't need a `word.split()`, you can just `join()` straight away.

Anyway, I've added a unit test that looks for `join` and `split` in the code of your function, no more cheating ;-)

Reply:
Fixed the "Cheating" :)